clients/spark/build.sbt

```diff
@@ -60,6 +60,9 @@ def generateCoreProject(buildType: BuildType) =
         // version, but ask to use whatever is provided so we do not
         // override what it selects.
         "com.amazonaws" % "aws-java-sdk-bundle" % "1.12.194" % "provided",
+        "com.azure" % "azure-core" % "1.10.0",
+        "com.azure" % "azure-storage-blob" % "12.9.0",
+        "com.azure" % "azure-storage-blob-batch" % "12.7.0",
```

Comment:
Assuming these will be "provided" as part of your work on fixing dependencies :-).  (I reckon if they are "provided", we can just `--package` them into `spark-submit` in the worst case.  I hope...)

Reply:
How can we rely on marking those as provided? I don't expect that every user who runs Spark on Azure will have those packages available.